packages/manager/modules/iam/src/components/deleteEntity/deleteEntity.controller.js

```diff
@@ -37,6 +37,49 @@ export default class DeleteEntityController {
     return !this.isDeleting && this.form?.$valid;
   }
 
+  /**
+   * A contextualized Set where each key map to a tag
+   * @returns {{
+   *  close: string;
+   *  delete: string;
+   *  error?: string;
+   *  success?: string;
+   * } | null}
+   */
+  get tag() {
+    if (this.entity.type === ENTITY.POLICY) {
```

Comment:
suggestion : once again, maybe you can put a global object (Map) in constants file just to have to call CONSTANTS_OBJECT[policy]